### ISSUE TITLE
Synchronizing file via WebDAV has problems if you'd already synced the file some other way

### ISSUE THREAD

**Issue by detrout** [orgzly/orgzly-android#632]
- [X] I have searched for existing issues that may be the same as or related to mine.

I had a file that I had previously synced via the nextcloud android app. I then added the same nextcloud folder via WebDAV. This caused the file to have two repositories linked to it. I tried to change what repository I was using but had an file name error in red text. (I think it was file name was null)

I eventually pointed the conflicting file at the repository in the nextcloud app directory, deleted it and it then downloaded from the WebDAV repository and everything was fine.

I suspect it's probably a good idea to sync your files, then remove files that had been synced by some other method before adding the WebDAV repository.

**Comment by nevenz** [comment]
Yeah, changing repositories is not handled well. You could have just changed the link and force-loaded the notebook, but you'd have to do this for each notebook. Supporting these actions for multiple notebooks would make things easier.

**Comment by detrout** [comment]
I think I tried to force download it (I pushed the cloud with down arrow) but couldn't because of some kind of file error. I think it was file name but it's been a couple of days so I might be remembering in correctly.

I have a different device I'll try to replicate it and see if I can catch what errors are actually shown.

**Comment by detrout** [comment]
Of course the other tablet worked perfectly.

Tablet where it worked perfectly was running Lineage OS. 15.1 (Android 8.10), nextcloud 3.8.1, orgzly  1.8.1

Phone that had trouble was running Samsung "One UI 1.0" (Android 9), nextcloud 3.8.1, orgzly 1.8.1
